test/e2e_test_insecure_registry.sh

```diff
@@ -1,46 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
oh, you removed this file but that is needed for the running test

Reply:
Yeah, but it was not testing weh policy-controller, but instead cosign against insecure registry, so I updated the action to not run this.